clients/src/main/java/org/apache/kafka/clients/ClientRequest.java

```diff
@@ -19,35 +19,35 @@
  */
 public final class ClientRequest {
 
-    private final long createdMs;
+    private final long createdTimeMs;
     private final boolean expectResponse;
     private final RequestSend request;
     private final RequestCompletionHandler callback;
     private final boolean isInitiatedByNetworkClient;
-    private long sendMs;
+    private long sendTimeMs;
 
     /**
-     * @param createdMs The unix timestamp in milliseconds for the time at which this request was created.
+     * @param createdTimeMs The unix timestamp in milliseconds for the time at which this request was created.
      * @param expectResponse Should we expect a response message or is this request complete once it is sent?
      * @param request The request
      * @param callback A callback to execute when the response has been received (or null if no callback is necessary)
      */
-    public ClientRequest(long createdMs, boolean expectResponse, RequestSend request,
+    public ClientRequest(long createdTimeMs, boolean expectResponse, RequestSend request,
                          RequestCompletionHandler callback) {
-        this(createdMs, expectResponse, request, callback, false);
+        this(createdTimeMs, expectResponse, request, callback, false);
     }
 
     /**
-     * @param createdMs The unix timestamp in milliseconds for the time at which this request was created.
+     * @param createdTimeMs The unix timestamp in milliseconds for the time at which this request was created.
      * @param expectResponse Should we expect a response message or is this request complete once it is sent?
      * @param request The request
      * @param callback A callback to execute when the response has been received (or null if no callback is necessary)
      * @param isInitiatedByNetworkClient Is request initiated by network client, if yes, its
      *                                   response will be consumed by network client
      */
-    public ClientRequest(long createdMs, boolean expectResponse, RequestSend request,
+    public ClientRequest(long createdTimeMs, boolean expectResponse, RequestSend request,
                          RequestCompletionHandler callback, boolean isInitiatedByNetworkClient) {
-        this.createdMs = createdMs;
+        this.createdTimeMs = createdTimeMs;
         this.callback = callback;
         this.request = request;
         this.expectResponse = expectResponse;
```

Comment:
Just noticed that the `toString` method has been incomplete for a while - do you want to fix that here as well?